env/test

```diff
@@ -1,7 +1,8 @@
 # Add Web3 url
```

Comment:
I think we decided purposely in the past not to commit testing keys. Im personally ok with it